services/hanke-service/src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -105,3 +105,5 @@ databaseChangeLog:
       file: db/changelog/changesets/037-add-delete-cascade-for-attachments.yml
   - include:
       file: db/changelog/changesets/038-add-attachment-content-tables.sql
+  - include:
```

Comment:
040-drop-column-organisaatioid-from-yhteystiedot should be included here.

Reply:
Yes it should, thank you. I added it.